release-plz.toml

```diff
@@ -0,0 +1,24 @@
+[workspace]
+dependencies_update = true                   # Wether to run `cargo update` in the release PR
+pr_name = "chore(main): release {{version}}" # template for the PR name
+pr_labels = ["release", "crates.io"]         # Set PR labels
+changelog_path = "./CHANGELOG.md"            # Set common changelog path for all crates
+changelog_update = false                     # Disable individual changelog updates for all packages.
+git_tag_enable = false                       # Disable individual tags for all packages.
+git_release_enable = false                   # Disable individual releases for all packages.
+semver_check = true                          # Enable API breaking changes checks with cargo-semver-checks.
```

Comment:
Should be false

Reply:
Fixed to `false` and updated RELEASE.md with `release-plz` link. I believe the other content is still valid.